canopen/pdo/base.py

```diff
@@ -415,8 +415,13 @@ def add_variable(self, index, subindex=0, length=None):
             if length is not None:
                 # Custom bit length
                 var.length = length
-            logger.info("Adding %s (0x%X:%d, %d bits) to PDO map",
-                        var.name, var.index, var.subindex, var.length)
+            # We want to see the bit fields within the PDO
+            offset = var.offset
+            byte_offset, bit_offset = divmod(offset, 8)
+            sb = byte_offset * 8 + bit_offset
+            eb = sb + var.length - 1
+            logger.info("Adding %s (0x%X:%d, %d bits, %d:%d) to PDO map",
```

Comment:
Maybe it could be clearer what the values mean. If we log start and end bits then we don't need to have length also.

Reply:
Maybe `"Adding %s (0x%X:%d) at bits %d - %d to PDO map"`?

Reply:
Sounds good.

Reply:
Done. :)